src/utils/geo-processing-services.js

```diff
@@ -203,9 +201,10 @@ export const setPrecalculatedSpeciesData = (attributes, setTaxaData) => {
   getPrecalculatedSpeciesData(AMPHIBIANS, attributes.amphibians).then(data => setTaxaData(data));
 };
 
-const getAreaName = (data, config) => {
-  if (!config) return null;
-  return config.subtitle ? `${data[config.name]}, (${data[config.subtitle]})` : data[config.name];
+const getAreaName = (data) => {
```

Comment:
I see that we are overriding the config here and maybe that is why it doesn't matter what we have there

Reply:
Added protected area case